docs/src/pages/demos/text-fields/ComposedTextField.js

```diff
@@ -17,10 +20,16 @@ const styles = theme => ({
 });
 
 class ComposedTextField extends React.Component {
+  labelRef = null;
```

Comment:
I think we should use the modern `React.createRef` API.

Reply:
We could but it more boilerplate (like in the TextField source).